albumentations/augmentations/bbox_utils.py

```diff
@@ -218,25 +218,39 @@ def convert_bbox_to_albumentations(bbox, source_format, rows, cols, check_validi
         ValueError: If in YOLO format all labels not in range (0, 1).
 
     """
-    if source_format not in {"coco", "pascal_voc", "yolo"}:
+    if source_format not in {
+        "coco",
+        "pascal_voc",
+        "yolo",
+        "xyxy",
+        "xywh",
+        "xywh_center_norm",
+        "xywh_norm",
+        "xywh_center",
+    }:
```

Comment:
Hi! Thanks for your PR.

We can describe all formats in global variable, like 
```
BBOX_FORMATS = [
        "coco",
        "pascal_voc",
        "yolo",
        "xyxy",
        "xywh",
        "xywh_center_norm",
        "xywh_norm",
        "xywh_center",
]
```
It is better, because you forget there `xyxy_norm`, but you use it at `albumentations/core/utils.py` in `preprocess` function

Reply:
Hi @Dipet ! Thank you for comment.

I deliberately did not put `xyxy_norm` in this line, because `xyxy_norm` is the same as `albumentations` format and I think this format should not be used at `convert_bbox_to_albumentations` and `convert_bbox_from_albumentations` in `albumentations/augmentations/bbox_utils.py`.

In fact, in `check_and_convert` function that is called at `preprocess` function in `albumentations/core/utils.py`, `albumentations` and `xyxy_norm` formats escape to not reach `convert_bbox_to_albumentations` and `convert_bbox_from_albumentations`.

However, I think it is better to describe these formats in global variable, so I do that.

thank you!